### PR TITLE
Implement metadata update and seeking on remote control

### PR DESCRIPTION
**Pull request by romandev**
Currently, even if playing the video on RemoteControl service, the
metadata(current time, duration) is not updated yet. We should implement
them into model and update them on View(such as seekbar and label text).

This patch includes the following things:
  - Time update
  - Seeking on SeekBar
  - Rename setPosition with setCurrentTime (in remote_control module)

Refs: #612

### CODE REVIEW

File: src/org/triple/banana/remote_control/RemoteControlService.java [hunk 1]
```diff
@@ -159,13 +166,12 @@ public void onBrightnessChanged(float value) {
...
-        float position = mViewModel.getEditor().getPosition();
+    public void onCurrentTimeChanged(double diffTime) {
+        double newTime = mViewModel.getEditor().getCurrentTime() + diffTime;
         mViewModel.getEditor().setControlsVisibility(true);
-        mViewModel.getEditor().setPosition(position + value);
+        mViewModel.getEditor().setCurrentTime(newTime);
         mViewModel.commit();
-        // FIXME(#589): Need to fix seekTo operation of MediaSession
-        // mMediaController.setPosition(ms);
+        mMediaController.setPosition((float) (newTime));
```
Comment (jdragonbae) [inline src/org/triple/banana/remote_control/RemoteControlService.java:174]: Can we call setPosition when the diff time is 0.0?

File: src/org/triple/banana/remote_control/RemoteControlService.java [hunk 2]
```diff
@@ -159,13 +166,12 @@ public void onBrightnessChanged(float value) {
     }
 
     @Override
-    public void onPositionChanged(float value) {
-        float position = mViewModel.getEditor().getPosition();
+    public void onCurrentTimeChanged(double diffTime) {
+        double newTime = mViewModel.getEditor().getCurrentTime() + diffTime;
```
Comment (jdragonbae) [inline src/org/triple/banana/remote_control/RemoteControlService.java:170]: Do we need check isLocked? 
`if (mViewModel.getData().getIsLocked()) return;`

File: src/org/triple/banana/remote_control/RemoteControlService.java (hunk 1 above)
Comment (romandev) [reply src/org/triple/banana/remote_control/RemoteControlService.java]: I introduced `onPositionChangeFinish()` and used it. Done.

File: src/org/triple/banana/remote_control/RemoteControlService.java (hunk 2 above)
Comment (romandev) [reply src/org/triple/banana/remote_control/RemoteControlService.java]: Done.

**Review by jdragonbae — APPROVED** [review]
lgtm

**Pull request merged**